src/lib/services/suggest-change-service.ts

```diff
@@ -0,0 +1,79 @@
+import { IUnleashConfig, IUnleashStores } from '../types';
```

Comment:
The service should move to enterprise

Reply:
The reason is here is to keep the convention (see SegmentService)

Reply:
I suspect this is an oversight, and represents technical debt that we should move to enterprise also. We do have a setup for services in enterprise. 

@ivarconr